.github/workflows/ci.yml

```diff
@@ -41,18 +41,16 @@ jobs:
       matrix:
         ember-version:
           [
+            ember-lts-2.4,
```

Comment:
I am positive we want to drop 2.4 from our support matrix ;)

Reply:
Temporary change, gone now.